src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -134,7 +134,7 @@
 data_vpc = network_stack.require_output("data_vpc")
 data_integrator_secgroup = data_vpc["security_groups"]["integrator"]
 
-framework = "docker" if edxapp_config.get_bool("use_docker") else "ansible"
+framework = edxapp_config.get("framework") or "ansible"
```

Comment:
Given the current level of adoption it probably makes sense to default to docker rather than ansible.

Reply:
Fair.